README.md

```diff
@@ -63,8 +63,8 @@
 ### Key Features <a name="key-features"></a>
 
 - **[User_can_add_score]**
-- **[User_can_see_score_on_the_webpack]**
-- **[Scores_are_saved_in_local_storage]**
+- **[User_can_see_score_on_the_webpage]**
+- **[Scores_are_saved_and_displayed_from_api]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make sure to include a license in your readme file, as for now the LICENSE link redirect to a 404 page